locust/runners.py

```diff
@@ -750,13 +772,15 @@ def start(
         self.spawn_rate = spawn_rate
 
         if self._users_dispatcher is None:
-            self._users_dispatcher = UsersDispatcher(
+            self._users_dispatcher = self.environment.dispatcher_class(
                 worker_nodes=list(self.clients.values()), user_classes=self.user_classes
             )
 
+        _user_count = self._users_dispatcher.get_target_user_count() if isinstance(user_count, dict) else user_count
```

Comment:
can you think of a better name than `_user_count`? This code is very hard to read.